recipes/brotlipy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "brotlipy" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "36def0b859beaf21910157b4c33eb3b06d8ce459c942102f16988cca6ea164df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
This package compiles code, so you should add the `toolchain` build requirement.